src/TraceEvent/TraceEventStacks.cs

```diff
@@ -239,8 +240,12 @@ public override StackSourceCallStackIndex GetCallerIndex(StackSourceCallStackInd
                     nextIndex += (int)threadIndex;
 
                     // Mark it as a broken stack, which come after all the indexes for normal threads and processes. 
-                    if (!ReasonableTopFrame(callStackIndex, threadIndex))
+                    // TODO Fix NOW Clr thread sample only has managed call stack. Do we have a better solution to differentiate it from normal call stack?
+                    if (!string.Equals(m_log.origin, EventPipeEventSource.EventPipe, StringComparison.OrdinalIgnoreCase)
```

Comment:
We should make up a property onlyManagedCodeStacks for this purpose instead of origin.